From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Fri, 12 Jul 2019 05:22:11 -0400
Subject: [PATCH] disable safe browsing reporting opt-in by default

Adapted from GrapheneOS Vanadium patch 0035-disable-safe-browsing-reporting-opt-in-by-default.patch for the Vandium browser port.

---
 components/safe_browsing/core/common/safe_browsing_prefs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/safe_browsing/core/common/safe_browsing_prefs.cc b/components/safe_browsing/core/common/safe_browsing_prefs.cc
index cbda8e4a120033b36280f3fae18c75a5f94278bf..7ff4f4734706cdcf24a42cde6271fc3c55bd105b 100644
--- a/components/safe_browsing/core/common/safe_browsing_prefs.cc
+++ b/components/safe_browsing/core/common/safe_browsing_prefs.cc
@@ -215,7 +215,7 @@ void RegisterProfilePrefs(PrefRegistrySimple* registry) {
   registry->RegisterBooleanPref(
       prefs::kSafeBrowsingSawInterstitialScoutReporting, false);
   registry->RegisterBooleanPref(
-      prefs::kSafeBrowsingExtendedReportingOptInAllowed, true);
+      prefs::kSafeBrowsingExtendedReportingOptInAllowed, false);
   registry->RegisterTimePref(
       prefs::kSafeBrowsingEsbProtegoPingWithTokenLastLogTime, base::Time());
   registry->RegisterTimePref(
